ksql-engine/src/test/java/io/confluent/ksql/codegen/CodeGenRunnerTest.java

```diff
@@ -84,9 +86,17 @@
     private InternalFunctionRegistry functionRegistry = new InternalFunctionRegistry();
     private GenericRowValueTypeEnforcer genericRowValueTypeEnforcer;
 
+
     @Before
     public void init() {
         metaStore = MetaStoreFixture.getNewMetaStore(functionRegistry);
+        // load substring function
+        new UdfLoader(metaStore,
+            TestUtils.tempDirectory(),
+            getClass().getClassLoader(),
+            value -> true, new UdfCompiler(), true)
```

Comment:
The `value -> true` suggests that we are blacklisting all UDFs in this test. Any particular reason why?

Reply:
yeah - it should be the other way around. Though it doesn't matter as it only applies to those that are loaded from the ext directory and in this case none of them are.